src/stats.c

```diff
@@ -45,13 +45,13 @@ get_rate_str(uint64_t dividend, uint64_t divisor, char str[6]) {
 
 	unsigned n = (dividend * 1000) / divisor;
 	if (n < 10) {
-		malloc_snprintf(str, sizeof(str), "0.00%u", n);
+		malloc_snprintf(str, 6, "0.00%u", n);
```

Comment:
This update to ```get_rate_str()``` should be moved back to the original diff that creates the function.